Sources/MapboxCoreNavigation/EHorizon/ElectronicHorizonDelegate.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+import MapboxNavigationNative
+
+/**
+ Electronic horizon delegate. Methods are called in the order specified. `didUpdatePosition` might be called
```

Comment:
This needs to be spelled out in full in order for jazzy to automatically link it:

```suggestion
 Electronic horizon delegate. Methods are called in the order specified. `didUpdatePosition(_:distances:)` might be called
```

or if the method gets renamed:

```suggestion
 Electronic horizon delegate. Methods are called in the order specified. `didUpdate(_:distances:)` might be called
```